package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
Looks like the package.json content got removed, you can paste one from a previous version and then ensure that the app still works

Reply:
resolved